src/plugin.js

```diff
@@ -183,7 +183,7 @@ class PlaylistMenuItem extends Component {
 
     // Video title
     const titleEl = document.createElement('cite');
-    const titleText = item.name || this.localize('Untitled Video');
+    const titleText = item.name || item.name === '' ? item.name : this.localize('Untitled Video');
```

Comment:
Perhaps the condition ought to be `typeof item.name === 'string'` instead, though? That seems to be the actual intent: if a string value for `name` is provided, use that; otherwise, fall back.

Reply:
Yeah, it's a minor difference, but seems to me that one condition is clearer than two. Thanks!

Reply:
Yeah, something like
```js
const titleText = typeof item.name === 'string' : item.name ? this.localize('Untitled Video');
```
would be clearest